util/make_timeline.py

```diff
@@ -284,8 +375,9 @@ def main(args):
     parser.add_argument('-rf', '--fight', type=int, help="Fight ID of the report to use. Defaults to longest in the report")
 
     # Log file arguments
-    parser.add_argument('-s', '--start', type=timestamp_type, help="Timestamp of the start, e.g. '12:34:56.789")
-    parser.add_argument('-e', '--end', type=timestamp_type, help="Timestamp of the end, e.g. '12:34:56.789")
+    parser.add_argument('-s', '--start', help="Timestamp of the start, e.g. '12:34:56.789")
```

Comment:
I assume you're still going to look into what's going on with start/end/timestamp_type as well? I know you mentioned you had commented this out for testing.

Reply:
Yes, although I'm a little more concerned about this than I am the rest of the changes. I had hoped to get through these updates without needing to completely familiarize with `argparse`, but I'm not seeing a clear path to enable validation as you had originally set it up. That most likely means I don't know enough, so I'll have to try to take some time this next week to dig through `argparse` and learn it properly.